src/index.css

```diff
@@ -25,3 +27,218 @@ code {
     'Courier New',
     monospace;
 }
+
+#root {
+  width: 100%;
+  height: 100vh;
+  background-color: #f5f6fa;
+}
+
+.main {
+  padding: 2% 5%;
+}
+
+.panel-bg {
+  width: 70%;
+  height: auto;
+  margin: 0 auto;
+  padding: 0 0 1% 0;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  background-color: #f3f3f3;
+}
+
+.panel-nav {
+  width: 100%;
+  height: 10vh;
+  background-color: #fff;
+  display: flex;
+}
+
+.panel-nav h1 {
+  padding-left: 5%;
+  width: 15rem;
+  height: 2.313rem;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--azure);
+  font-family: Montserrat, sans-serif;
+}
+
+.nav-ul {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  gap: 5em;
+}
+
+.head {
+  margin-left: auto;
+  padding: 5%;
+}
+
+a {
+  text-decoration: none;
+}
```

Comment:
- [ ] I recommend creating a class to give the correct styles to the title and complete al the required changes by the last CR to approve this milestone. 👇🏽 

> Please make sure you are using the correct font family, Montserrat, and the accurate color, #0290ff. Could you also use the right color for the navigation links to match the design guideline? Please reduce the height of your navigation bar. It looks a bit bigger when compared to the design. 😎️